make/migrations/postgresql/0050_2.2.0_schema.up.sql

```diff
@@ -455,3 +455,58 @@ BEGIN
     END LOOP;
 END $$;
 
+/*migrate robot_token_duration from minutes to days if exist*/
+DO $$
+DECLARE
+   properties_info record;
+   duration_in_minutes text;
+   duration_in_days integer;
+BEGIN
+  SELECT INTO properties_info * FROM properties WHERE k = 'robot_token_duration';
+  IF properties_info IS NOT NULL THEN
+    duration_in_minutes = properties_info.v;
+    duration_in_days = cast(duration_in_minutes as integer) / 60 / 24;
+    update properties set v = cast(duration_in_days as text)  WHERE k = 'robot_token_duration';
+  END IF;
+END $$;
+
+/*
+Common vulnerability reporting schema.
+Github proposal link : https://github.com/goharbor/community/pull/145
+*/
+
+-- --------------------------------------------------
+--  Table Structure for `main.VulnerabilityRecord`
+-- --------------------------------------------------
+CREATE TABLE IF NOT EXISTS "vulnerability_record" (
+    "id" serial NOT NULL PRIMARY KEY,
+    "cve_id" text NOT NULL DEFAULT '' ,
+    "registration_uuid" text NOT NULL DEFAULT '',
+    "digest" text NOT NULL DEFAULT '' ,
+    "report_uuid" text NOT NULL DEFAULT '' ,
+    "package" text NOT NULL DEFAULT '' ,
+    "package_version" text NOT NULL DEFAULT '' ,
+    "package_type" text NOT NULL DEFAULT '' ,
+    "severity" text NOT NULL DEFAULT '' ,
+    "fixed_version" text,
+    "urls" text,
+    "cve3_score" double precision,
+    "cve2_score" double precision,
+    "cvss3_vector" text,
+    "cvss2_vector" text,
+    "description" text,
+    "cwe_ids" text,
+    "vendorattributes" json,
+    UNIQUE ("cve_id", "registration_uuid", "package", "package_version", "digest")
```

Comment:
We should not store artifact digest in this table.

```suggestion
    UNIQUE ("cve_id", "registration_uuid", "package", "package_version")
```